### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kpmmmurphy**

### CODE REVIEW

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java [hunk 1]
```diff
@@ -62,6 +63,10 @@
      * Flag to ensure we only set the margins once
      */
     private boolean marginSet;
+    /**
+     * Flag to enable / disable haptic feedbacl
```
Comment (AJScanlan) [inline alerter/src/main/java/com/tapadoo/alerter/Alert.java]: typo in feedback

File: alerter/src/main/java/com/tapadoo/alerter/Alerter.java [hunk 2]
```diff
@@ -355,6 +367,33 @@ public Alerter setOnHideListener(@NonNull final OnHideAlertListener listener) {
     }
 
     /**
+     * Enable Vibration
```
Comment (AJScanlan) [inline alerter/src/main/java/com/tapadoo/alerter/Alerter.java]: Maybe just specify it's on be default

File: alerter/src/main/res/layout/alerter_alert_view.xml [hunk 3]
```diff
@@ -1,77 +1,85 @@
...
-        android:id="@+id/rlContainer"
-        android:layout_width="wrap_content"
-        android:layout_height="match_parent"
-        android:layout_gravity="center_vertical"
-        android:layout_marginTop="@dimen/alerter_padding_half"
-        android:background="@null"
-        android:clipChildren="false"
-        android:clipToPadding="false"
-        android:padding="@dimen/alerter_activity_horizontal_margin">
+    <FrameLayout
```
Comment (AJScanlan) [inline alerter/src/main/res/layout/alerter_alert_view.xml]: This framelayout and the one above it have different ordering of attributes. Very minor but can be nice to have everything using a consistent order

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java (hunk 1 above)
Comment (kpmmmurphy) [inline alerter/src/main/java/com/tapadoo/alerter/Alert.java]: Good catch.

File: alerter/src/main/java/com/tapadoo/alerter/Alerter.java (hunk 2 above)
Comment (kpmmmurphy) [inline alerter/src/main/java/com/tapadoo/alerter/Alerter.java]: It's on by default, I'll update the docs to make this clear.

File: alerter/src/main/res/layout/alerter_alert_view.xml (hunk 3 above)
Comment (kpmmmurphy) [inline alerter/src/main/res/layout/alerter_alert_view.xml]: Haha, you're a harsh critic Alex. I'll reformat.

**Pull request merged**